src/main/java/anab/Person/ManagePerson.java

```diff
@@ -2,8 +2,23 @@
 
 public class ManagePerson {
 
+    static void printeazaText(String name) {
+        System.out.println("Hello!" + name);
+    }
+
     public static void main(String[] args) {
-        Person person = new Person();
+        Person personPerson = new Person("Ion", "Popescu", "Gender", 15, "EmailAddress");// create an object of Person class
```

Comment:
why personPerson and not simply person ?

Reply:
Was easier for me to see the difference between Constructor and the object initialized.